app/controllers/static_pages_controller.rb

```diff
@@ -4,4 +4,6 @@ def home; end
   def help; end
 
   def about; end
+
+  def contact; end
```

Comment:
empty methods doesn't make sense.
so if you remove them and keep only routes - it will work